### DISCUSSION TITLE
Documentation confusing with Spectre.Console.Cli

### DISCUSSION THREAD

**Discussion by NicolasR** [spectreconsole/spectre.console#1397] [Q&A] [answered]
Hello,

I wanted to use Spectre CLI and i struggled a little to understand what i need to install to use CLI.

According to documentation [https://spectreconsole.net/cli/migration](https://spectreconsole.net/cli/migration), i need to install Spectre.Console to be able to use CLI. 
However CLI is on its own package: Spectre.Console.Cli, which is not Spectre.Console. 

Did i miss something or documentation is really confusing ?

**Comment by patriksvensson** [comment, marked as answer]
That documentation is for migrating from the old `Spectre.Cli` package to `Spectre.Console.Cli`.

The [Getting started](https://spectreconsole.net/cli/getting-started) section is what you're looking for.
